### DISCUSSION TITLE
为什么克隆下来下载依赖netty依赖无法加载?我尝试谷歌但是无法解决😂,请大佬指点

### DISCUSSION THREAD

**Discussion by aozeyu** [Ahoo-Wang/CosId#191] [Q&A] [answered]
<img width="1238" alt="截屏2022-10-29 10 17 54" src="https://user-images.githubusercontent.com/55688386/198772830-c561ab59-15fa-4afd-98a6-9631eb03802e.png">
谷歌的解决方法是如下图但是我加了也没有效果
<img width="1253" alt="截屏2022-10-29 10 18 16" src="https://user-images.githubusercontent.com/55688386/198773091-d1008f9b-4ef7-466b-a488-b7c8d955ca69.png">

**Comment by Ahoo-Wang** [comment, marked as answer]
最直接的办法将
https://repo1.maven.org/maven2/io/netty/netty-transport-native-epoll/4.1.84.Final/
下载到 
![image](https://user-images.githubusercontent.com/4384159/198780306-1be72cb8-eb8b-4191-9c84-6c780da77311.png)
该目录

**Reply by aozeyu** [reply]
好的感谢回复,但是为什么会出现这个问题呢是gradle的问题吗,因为也有人遇到了和我一样的问题,还是说4.1.84这个版本的问题,必须手动下载呢。
<img width="1024" alt="截屏2022-10-29 10 37 40" src="https://user-images.githubusercontent.com/55688386/198782087-fcdae43a-5c5c-4003-92c7-3f2d58cd3903.png">
